generate.ts

```diff
@@ -135,10 +139,10 @@ function parseCommandInputs(args, argsByName, errorMessages, targetOutputPathLis
         sdkGeneratorGlobals.unitySubfolder = argsByName.unityDestinationSubfolder;
 
     // Output an error if no targets are defined
-    if (targetOutputPathList.length === 0)
-        errorMessages.push("No targets defined, you must define at least one.");
+    if (!buildTarget.destPath)
+        errorMessages.push("Build target not defined, nothing to build.");
```

Comment:
>Build target [](start = 28, length = 12)

"Build target's destPath not defined", it's probably better to be more precise with error messages